Jint/Runtime/Environments/DeclarativeEnvironmentRecord.cs

```diff
@@ -78,14 +73,14 @@ public override void SetMutableBinding(in Key name, JsValue value, bool strict)
 
         public override JsValue GetBindingValue(in Key name, bool strict)
         {
-            ref var binding = ref GetOrCreateBinding(name);
+            _dictionary.TryGetValue(name, out var binding);
             return UnwrapBindingValue(strict, binding);
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private JsValue UnwrapBindingValue(bool strict, in Binding binding)
         {
-            if (!binding.Mutable && !binding.IsInitialized)
```

Comment:
I'd keep IsInitialized as it shouldn't matter for perf and is easier to read the spec

Reply:
Returned the old version